.github/CODEOWNERS

```diff
@@ -6,30 +6,37 @@
 ################
 
 # Git Hub integration and bot rules
-/.github/                @AlexGhiondea @jsquire
+/.github/                @AlexGhiondea @jsquire @rickwinter
 
 ###########
 # SDK
 ###########
 
 # Catch all for non-code project files and unowned files | folders
-*                        @rickwinter @ahsonkhan @antkmsft @vhvb1989 @gearama
-/sdk/                    @rickwinter @ahsonkhan @antkmsft @vhvb1989 @gearama
+*                        @rickwinter @ahsonkhan @antkmsft @vhvb1989 @gearama @larryo
+/sdk/                    @rickwinter @ahsonkhan @antkmsft @vhvb1989 @gearama @larryo
```

Comment:
Do we have a team that we can use here?  like @Azure/cppTeam instead of letting the list go bigger and bigger

Reply:
We don't currently.  I'll get us one.